test/cloudtest/pkg/providers/packet/packet_provider.go

```diff
@@ -102,12 +101,9 @@ func (pi *packetInstance) Start(timeout time.Duration) error {
 
 	utils.ClearFolder(pi.root, true)
 
-	today := time.Now()
-	pi.genID = fmt.Sprintf("%d-%d-%d-%s", today.Year(), today.Month(), today.Day(), shell.NewRandomStr(10))
 	// Process and prepare environment variables
 	if err = pi.shellInterface.ProcessEnvironment(
-		pi.id, pi.config.Name, pi.root, pi.config.Env,
-		map[string]string{"cluster-uuid": pi.genID}); err != nil {
+		pi.id, pi.config.Name, pi.root, pi.config.Env, map[string]string{}); err != nil {
```

Comment:
Since in `ProcessEnvironment` we have:
```go
for k, v := range extraArgs {
	args[k] = v
}
```
you can simply pass `nil` instead of `map[string]string{}`